server/auth/index.js

```diff
@@ -2,22 +2,27 @@ const passport = require('koa-passport');
 
 const oidc = require('./oidc');
 const { STRATEGY_NAMES } = require('./constants');
-const { isAuthEnabled, getAuthConfig } = require('../config');
+const { getAuthConfig } = require('../config');
 
 const initialize = async (ctx, next) => {
   if (!passport._strategies[STRATEGY_NAMES.oidc]) {
-    const enabled = await isAuthEnabled();
+    const enabled = (await getAuthConfig()).enabled;
     if (!enabled) {
       throw Error('No authentication configuration found');
     }
```

Comment:
why are we throwing if false? shouldnt we just disable auth? is this a TODO for a future PR or should we fix this in this PR? ultimately lets make sure we dont break existing (no auth) behavior